presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,124 @@
+* Add session property: ``native_max_page_partitioning_buffer_size`` which specifies the maximum bytes to buffer per PartitionedOutput operator to avoid creating tiny SerializedPages. :pr:`23853`
+* Add session property: ``native_max_output_buffer_size`` which specifies the maximum size in bytes for the task's buffered output. The buffer is shared among all drivers. :pr:`23853`
+* Add incremental periodic cache persistence for Presto C++ worker. :pr:`23626`
+* Add native system session property provider. :pr:`23045`
+* Remove session property ``native_join_spiller_partition_bits``. :pr:`23906`
+
+Security Changes
+________________
+* Upgrade Postgres JDBC Driver to 42.6.1. :pr:`23710`
+* Upgrade the logback-core version to 1.2.13. :pr:`23735`
+
+Hive Connector Changes
+______________________
+* Fix interpretation of ambiguous timestamps inside array, map, or row types for tables using ``TEXTFILE`` format to interpret the timestamps as the earliest possible unixtime for consistency with the rest of Presto. :pr:`23593`
+* Fix timestamps inside array, map, or row types for tables using ``TEXTFILE`` format to respect the ``hive.time-zone property``. :pr:`23593`
+
+Iceberg Connector Changes
+_________________________
+* Fix time-type columns to return properly when ``iceberg.parquet-batch-read-optimization-enabled`` is set to ``TRUE``. :pr:`23542`
+* Fix to reduce drop time for Iceberg tables with deleted metadata in S3 storage. :pr:`23510`
+* Add Iceberg metadata table ``$ref``. :pr:`23503`
+* Add configuration property ``iceberg.rest.auth.oauth2.scope`` for OAUTH2 authentication in Iceberg's REST catalog. :pr:`23884`
+* Add configuration property ``iceberg.rest.auth.oauth2.uri``. :pr:`23739`
+* Add procedure ``rollback_to_timestamp`` to rollback an Iceberg table to a given point in time. :pr:`23559`
+* Add support of UUID-typed columns. :pr:`23627`
+* Add support to query Iceberg table by branch/tag name. :pr:`23539`
+* Add table property ``metrics_max_inferred_column`` to configure the max columns number for which metrics are collected, and support ``metrics_max_inferred_column`` for Iceberg tables with `PARQUET` format. :pr:`23468`
+* Add support for procedure ``fast_forward`` for Iceberg. :pr:`23589`
+* Add support for using named arguments in procedures ``register_table`` and ``unregister_table``. :pr:`23592`
+* Support new procedure ``set_current_snapshot`` for Iceberg. :pr:`23567`
+* Support timestamp without timezone in time travel expressions. :pr:`23714`.
+
+MongoDB Connector Changes
+_________________________
+* Add support for ``varbinary`` data type in MongoDB. :pr:`23386`
+* Add support for MongoDB ``ALTER TABLE`` statement. :pr:`23266`
+
+Cassandra Connector Changes
+___________________________
+* Upgrade cassandra-driver-core to 3.11.5 for SSL support. :pr:`23493`
+
+Elasticsearch Connector Changes
+_____________________
```

Comment:
```suggestion
_______________________________
```
I should have caught this when I suggested changing the text in line 105 from `Elasticsearch Changes` to `Elasticsearch Connector Changes` - the length of the formatting line and the text it formats should match.